cppcache/src/PdxTypeRegistry.cpp

```diff
@@ -89,8 +89,8 @@ int32_t PdxTypeRegistry::getPDXIdForType(std::shared_ptr<PdxType> nType,
     typeId = cache_->getSerializationRegistry()->GetPDXIdForType(pool, nType);
     nType->setTypeId(typeId);
     pdxTypeToTypeIdMap_.insert(std::make_pair(nType, typeId));
+    typeIdToPdxType_.insert(std::make_pair(typeId, nType));
   }
-  addPdxType(typeId, nType);
```

Comment:
Perhaps it makes sense to remove this method and inline the single use in the `getPDXIdForType` function.

Reply:
`PdxTypeRegistry::addPdxType()` is used several times in `PdxHelper` class, I cannot be removed. It is also used other time in `PdxTypeRegistry` class, and I think it has sense to keep it for readability: it cannot be replace just by a `typeIdToPdxType_.emplace()` call because it is also needed to acquire the lock, which is done inside `addPdxType()`